spec/models/group_engagement_scopes_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+describe Group do
+  describe 'engagement-scopes' do
+    describe 'more_than_n_members' do
+      let(:group_with_no_members) { FactoryGirl.create :group }
```

Comment:
yea... they're for the past. I'll delete them. They were more a way of building engagement scopes correctly rather than keeping things tested